src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1198,12 +1198,12 @@ public final class JavadocTokenTypes {
     /** `rb` tag name. */
     public static final int RB_HTML_TAG_NAME = JavadocParser.RB_HTML_TAG_NAME;
 
-    /** `rt` tag name. */
-    public static final int RT_HTML_TAG_NAME = JavadocParser.RT_HTML_TAG_NAME;
-
     /** `rtc` tag name. */
     public static final int RTC_HTML_TAG_NAME = JavadocParser.RTC_HTML_TAG_NAME;
 
+    /** `rt` tag namw. */
```

Comment:
```suggestion
    /** `rt` tag name. */
```

Why did you changed this variable's position? I don't think we needed it

Reply:
just changed debug  if this particular statement is getting flagged or that particular line no. by antrun.
earlier 1201 was getting flagged after changing the position still 1201 is getting flagged. 

pls ignore my typo for now.